activestorage/README.md

```diff
@@ -2,10 +2,10 @@
 
 Active Storage makes it simple to upload and reference files in cloud services, like Amazon S3, Google Cloud Storage or Microsoft Azure Storage and attach those files to Active Records. It also provides a disk service for testing or local deployments, but the focus is on cloud storage.
 
-Files can uploaded from the server to the cloud or directly from the client to the cloud.
+Files can be uploaded from the server to the cloud or directly from the client to the cloud.
 
 Image files can further more be transformed using on-demand variants for quality, aspect ratio, size, or any other
-MiniMagick supported transformation.
+`MiniMagick` supported transformation.
```

Comment:
:-1:

“MiniMagick” refers to the library here, not the `MiniMagick` constant.

(There should be a hyphen between “MiniMagick” and “supported,” though.)